### ISSUE TITLE
PostgreSQL errors

### ISSUE THREAD

**Issue by lavrin** [esl/MongooseIM#369]
MongooseIM causes PostgreSQL to log a lot of these:

```
2015-02-06 00:33:49 GMT [11783]: [110-1] user=ejabberd,db=ejabberdHINT: Use the escape string syntax for backslashes, e.g., E'\'.
2015-02-06 00:33:49 GMT [11779]: [127-1] user=ejabberd,db=ejabberdWARNING: nonstandard use of \ in a string literal at character 229
```

Does anyone know anything useful? Any fix?

**Comment by michalwski** [comment]
This was fixed in #275, to be more specific in [this line](https://github.com/esl/MongooseIM/pull/275/files#diff-135033bebc27a9b61ccfe95fa66beb81R778).

**Comment by lavrin** [comment]
Thanks, I'll have to do some backporting.
